src/publisher/api_v2_views.py

```diff
@@ -107,16 +110,18 @@ def article_version(request, id, version):
 
 @api_view(['POST'])
 def article_fragment(request, art_id, fragment_id):
-    only_published = is_authenticated(request)
+    if not is_authenticated(request):
+        return Response("not authenticated. only authenticated admin users can modify content", status=403)
     try:
         reserved_keys = [XML2JSON]
         ensure(fragment_id not in reserved_keys, "that key is taken")
         with transaction.atomic():
-            av = logic.most_recent_article_version(art_id, only_published)
-            data = request.data
-            frag, created, updated = fragment_logic.add(av.article, fragment_id, data, update=True)
-            ensure(created or updated, "fragment was not created/updated")
-            fragment_logic.merge_if_valid(av, quiet=False)
+            avs = logic.article_version_list(art_id, only_published=False)
+            for av in avs:
```

Comment:
the current behavior is then that if a `vN` article version is published, the article "loses" the fragment because it's attached to `v1`, ..., `v(N-1)` only. I was wondering if there was a way to attach it to the article, but since it seems there is not a way to do that, we need a Jira ticket.